playground/nextjs-app-router/components/Demo.tsx

```diff
@@ -45,66 +62,9 @@ function Demo() {
       : 'border-gray-300 bg-white text-black hover:bg-gray-100'
   }`;
 
-  // biome-ignore lint/complexity/noExcessiveCognitiveComplexity: TODO: refactor
-  function renderActiveComponent() {
-    if (activeComponent === OnchainKitComponent.Fund) {
-      return <FundDemo />;
-    }
-
-    if (activeComponent === OnchainKitComponent.Identity) {
-      return <IdentityDemo />;
-    }
-
-    if (activeComponent === OnchainKitComponent.Transaction) {
-      return <TransactionDemo />;
-    }
-
-    if (activeComponent === OnchainKitComponent.Checkout) {
-      return <CheckoutDemo />;
-    }
-
-    if (activeComponent === OnchainKitComponent.Swap) {
-      return <SwapDemo />;
-    }
-
-    if (activeComponent === OnchainKitComponent.SwapDefault) {
-      return <SwapDefaultDemo />;
-    }
-
-    if (activeComponent === OnchainKitComponent.Wallet) {
-      return <WalletDemo />;
-    }
-
-    if (activeComponent === OnchainKitComponent.WalletDefault) {
-      return <WalletDefaultDemo />;
-    }
-
-    if (activeComponent === OnchainKitComponent.TransactionDefault) {
-      return <TransactionDefaultDemo />;
-    }
-
-    if (activeComponent === OnchainKitComponent.NFTMintCard) {
-      return <NFTMintCardDemo />;
-    }
-
-    if (activeComponent === OnchainKitComponent.NFTCard) {
-      return <NFTCardDemo />;
-    }
-
-    if (activeComponent === OnchainKitComponent.NFTMintCardDefault) {
-      return <NFTMintCardDefaultDemo />;
-    }
-
-    if (activeComponent === OnchainKitComponent.NFTCardDefault) {
-      return <NFTCardDefaultDemo />;
-    }
-
-    if (activeComponent === OnchainKitComponent.IdentityCard) {
-      return <IdentityCardDemo />;
-    }
-
-    return <></>;
-  }
+  const ActiveComponent = activeComponent
+    ? activeComponentMapping[activeComponent]
+    : null;
```

Comment:
nit: could eliminate your null check below and just return a component that renders null